### PR TITLE
[FIX] business_requirement_deliverable_report

### PR DESCRIPTION
**Pull request by serpentcs-dev1**
## Business Requirement Deliverable Report

Add some of the features of V9 branch within v8 reports

- Added Attachment Feature
- And adjusted Report as per Paperformat

@victormartinelicocorp @elicoidal

### CODE REVIEW

File: business_requirement_deliverable_report/data/br_report_paperformat.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+    <record id="br_report_paperformat_us" model="report.paperformat">
+        <field name="name">BR Report US Letter</field>
```
Comment (elicoidal) [inline business_requirement_deliverable_report/data/br_report_paperformat.xml]: Better by default European A4

**Comment by elicoidal** [comment]
@darshan-serpent the paper format is way better but you need to take back the PR #139 for all other improvements and fix it with newer fields and structure.

**Review by elicoidal — CHANGES_REQUESTED** [review]

**Comment by elicoidal** [comment]
Can you please check travis?

**Comment by serpentcs-dev1** [comment]
@elicoidal Fixed the travis, Please review.

**Comment by elicoidal** [comment]
@darshan-serpent you are fixing the wrong PR.
We need to get the module fixed in v9 and that @victormartinelicocorp has pushed in #139 
Then we can add your improvements

**Comment by serpentcs-dev1** [comment]
@elicoidal Sorry for the trouble. Okay I'll add this changes at the right place.

**Comment by serpentcs-dev1** [comment]
Closing the PR as the functionality is already added to #139

**Pull request closed without merging**